docs/functions/Bundles/Start-UPloadVCFBundle.md

```diff
@@ -1,38 +0,0 @@
-# Start-UploadVCFBundle
-
-### Synopsis
-The Start-UploadVCFBundle starts upload of bundle to SDDC Manager
```

Comment:
Module Name should be 'Start-VCFUploadBundle'

Reply:
same as above. It's a deleted file